pkgs/top-level/all-packages.nix

```
@@ -2601,15 +2601,47 @@ with pkgs;
 
   go-audit = callPackage ../tools/system/go-audit { };
 
-  gopass = callPackage ../tools/security/gopass { };
+  gopass = callPackage ../tools/security/gopass {
+    # pinned due to https://github.com/gopasspw/gopass/issues/2280
+    # when go_1_18 is bumped to 1.18.5 please remove __buildGo1183Module
+    buildGoModule = __buildGo1183Module;
+  };
+
+  gopass-hibp = callPackage ../tools/security/gopass/hibp.nix {
+    # pinned due to https://github.com/gopasspw/gopass/issues/2280
+    # when go_1_18 is bumped to 1.18.5 please remove __buildGo1183Module
+    buildGoModule = __buildGo1183Module;
+  };
 
-  gopass-hibp = callPackage ../tools/security/gopass/hibp.nix { };
+  gopass-jsonapi = callPackage ../tools/security/gopass/jsonapi.nix {
+    # pinned due to https://github.com/gopasspw/gopass/issues/2280
+    # when go_1_18 is bumped to 1.18.5 please remove __buildGo1183Module
+    buildGoModule = __buildGo1183Module;
+  };
 
-  gopass-jsonapi = callPackage ../tools/security/gopass/jsonapi.nix { };
+  git-credential-gopass = callPackage ../tools/security/gopass/git-credential.nix {
+    # pinned due to https://github.com/gopasspw/gopass/issues/2280
+    # when go_1_18 is bumped to 1.18.5 please remove __buildGo1183Module
+    buildGoModule = __buildGo1183Module;
+  };
 
-  git-credential-gopass = callPackage ../tools/security/gopass/git-credential.nix { };
+  gopass-summon-provider = callPackage ../tools/security/gopass/summon.nix {
+    # pinned due to https://github.com/gopasspw/gopass/issues/2280
+    # when go_1_18 is bumped to 1.18.5 please remove __buildGo1183Module
+    buildGoModule = __buildGo1183Module;
+  };
 
-  gopass-summon-provider = callPackage ../tools/security/gopass/summon.nix { };
+  # go 1.18 requires a newer Apple SDK to be build. See commit message for more details.
```

doesn't make sense, commit message doesn't include this detail.
```suggestion
  # go 1.18 requires a newer Apple SDK to be build.
```